src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
The first part of my last comment was:
> I always tried to directly refer to your previous comments and not introduce new topics into the discussion. Can you please give examples of posts in this thread where I did not refer to your comments to support your claim and if so to help me avoid that in the future? The other way around I find that you mentioned topics like escaping captures/SEE (ok, at least partly related), ttcaptures, testing conditions/bounds, other engines/search techniques, atomic chess, "single mistake" issues, upstream changes, etc. So maybe I ask new questions because you introduce new topics?

If I understand correctly, the following part of your comment is your response to that.
> We explored several questions here:
> 
> What's my idea regarding next steps (on anti_lmr7 pass/fail)? (I don't know what would be uncovered during cause analysis.)
> What's my motivation (why is it important) for searching for a similar patch?
> Anyway...

Considering that I complained about dodging questions, you can not really have expected that I'll be happy about this kind of reply.


> Sorry, I wasn't quoting, I just didn't have a term for our earlier difference in perspective & while I see positive results (both Elo and scaling) at present I don't know & am trying to understand it before making a value judgment. Even earlier (my plan based on anti_lmr7 results) didn't state anything about rejecting a patch (but that I intended to do further analysis to try to better understand this patch) & I had also stated I'd likely merge this PR. Anyway, I guess we'll see how next steps go...

Yes, saying that it would be rejected was an overstatement, that was just the way I interpreted the postponing of the decision in case of a negative test result when the criteria for the decision seemed to be undefined.